internal/testing/cmd/ingest/cmd/root.go

```diff
@@ -85,7 +87,9 @@ var rootCmd = &cobra.Command{
 	Use:   "ingest",
 	Short: "example ingestor for ingesting a set of example documents and populating a graph for GUAC",
 	Run: func(cmd *cobra.Command, args []string) {
-		ingestExample(cmd, args)
+		httpClient := http.Client{}
```

Comment:
we may need to refactor this. We don't want to be querying graphQL on ingestion. Will need to re-work this after some thought.

Reply:
Will make it an issue to discuss during the next maintainer meeting.